pylint/pyreverse/main.py

```diff
@@ -77,6 +78,8 @@
         dict(
             short="A",
             default=None,
+            type="yn",
```

Comment:
Sorry if I maybe was not clear enough on the other review.
``A`` and ``S`` are flags, that are used like ``pyreverse -AS <mypackage>``. Changing the type to "yn" is incompatible with the current usage.

Reply:
Should they be changed to action `store_true` then?

Reply:
This would definitely make sense, but we have to change the code as well. If you look at the help message for the ``class`` option you can see that the ``class`` option has a coupling with ``A``, ``S`` and ``m`` and the default value ``None`` (see ``DiaDefGenerator._set_default_options`` and the calls to ``_set_option``).

Reply:
Yeah I saw that, but I think `store_true` should work. As long as `default` is not a string the `type` (this time the type of `store_true`) won't be called so they would still be `None`.